src/clientSecret.ts

```diff
@@ -0,0 +1,4 @@
+export type CLIENTSECRET = {
```

Comment:
any reason for all caps?
```suggestion
export type ClientSecret = {
```